phpstan/stubs/apcu.php

```
@@ -0,0 +1,321 @@
+<?php
```

would it make sense to install these stubs via composer instead of copying them into the repos (that way those would automatically get upstream fixes/updates)

---

You cannot install php extensions via composer.
I tried to install APCu into travis environment but failed. Feel free to add ;-)

---

I am not saying we should try to install the extensions, but instead of installing them, always use the stubs

---

Proper installation of php extension into CI makes environment more production like. Stubs can be out of date, invalid, incomplete etc.

As pointed below, we can decide if we want to use stubs or ignore errors (and possible issues) 
from rarely used extension completly.

---

IMO ignoring is fine.. no need for stub or installing these. These exts are hardly used.